typings/index.d.ts

```diff
@@ -1632,6 +1627,8 @@ export class MessageActionRow<
     ? APIActionRowComponent<APIModalActionRowComponent>
     : APIActionRowComponent<APIMessageActionRowComponent>,
 > extends BaseMessageComponent {
+  // @ts-expect-error (TS:2344, Caused by TypeScript 4.8)
```

Comment:
Ig `ts-expect-error` will throw for ts<4.8 users as it's valid.  Instead use `ts-ignore` here

Reply:
Good point. Resolved.